.travis.yml

```diff
@@ -21,5 +21,5 @@ script:
   - sh -x ./node_modules/patternfly-eng-release/scripts/_build.sh -o
 
 after_success:
-  - grunt build
+  - grunt build:production
```

Comment:
The release still needs to build because we do a lot after the build completes. We modify files first, then build, shrinkwrap, verify installs, publish npm, push to master, master-dist, etc. Adding the build step in Travis config would introduce a timing issue.

This would be a problem for a non-release build as well. For most repos, we also push generated files to master-dist.  We don't have a master-dist for pf.org; however, It's easier if the scripts do this because they decide if we're performing a release build or not and publish to master-dist.